### ISSUE TITLE
Cadian Battle Group addition

### ISSUE THREAD

**Issue by BSDataAnon** [BSData/wh40k-7th-edition#1642]
**File:** Astra Militarum - Codex.catz

**BattleScribe version:** 1.14.07

**Platform:** Android

**Dropbox:** Yes

**Description:** Im trying to use the Astra militarum codex for the cadian battlegroup detachment, but it doesn't work with it, i would like to know if im using the right codex or if something is wrong with my update?

**Comment by ghost** [comment]
+1. No units listed. Is there another issue tracking this? I could not find one.
- [ ] Add existing Codex: Astra Militarum units to Cadian Battlegroup
- [ ] Create Formations for Cadian Battlegroup
- [ ] Input Formation rules for Cadian Battlegroup 
- [ ] Input Detachment special rules for Cadian Battlegroup

**Comment by ecrousseau** [comment]
I have created a catalogue file for the Cadian Battle Group - just waiting on permission to commit it.

**Comment by ghost** [comment]
@ecrousseau Cool! Was there a pull request pulled in?

**Comment by amis92** [comment]
Citing from duplicate issue:

> Would that be you too? https://www.reddit.com/r/BattleScribe/comments/3v8j0n/ive_created_a_catalogue_file_for_the_cadian/
> 
> As KeaIL asked: if it's new catalogue, separate from the existing one, we'll probably not consider it. These options will have to be incorporated into the main file. Of course, the last saying goes to maintainers, which currently looks to be @axisofentropy (or am I wrong?).

**Comment by ghost** [comment]
That's not me. :)

**Comment by amis92** [comment]
I meant @ecrousseau :D

**Comment by ecrousseau** [comment]
Yes that is my reddit post. I did make it a separate catalogue file, because I wanted to keep a 'light touch'. I believe it'd be pretty easy to merge into the main AM book if that is what people prefer, though? 

Edit: merging it into Codex: AM does make sense, so I'm doing that now.

**Comment by ecrousseau** [comment]
I'm done with the catalogue file. I'm not very familiar with git, hopefully you can tell me where I'm going wrong. Here's what I've done so far:
1. Install the GitHub software, and synced to the master branch
2. Created my own branch, and made my changes to those files

From there, my understanding is that I need to create a "pull request" - do I do that in my local git software, or here on the site, or...? There is a button for it in the GitHub software, but it gives me a permissions error.

Edit: I think I've figured it out... we'll know in a few minutes!

**Comment by ecrousseau** [comment]
Ok. Got it: pull request submitted :)

**Comment by zurran** [comment]
how long does this take to update from pull request to usable?

**Comment by epvanhouten** [comment]
It has to be merged in and then someone has to push a release.
